patches/chrome-android-java-src-org-chromium-chrome-browser-sync-settings-ManageSyncSettings.java.patch

```diff
@@ -17,6 +17,6 @@ index 0e09dbed0837ec74dd3fef73f3d92555faa7156c..e76992f5ae4e9d92cc94c32ac6fc3efc
          // May happen if account is removed from the device while this screen is shown.
 -        if (signedInAccountName == null) {
 +        if (false) {
-             if (getActivity() != null) getActivity().finish();
+             finishCurrentSettings();
```

Comment:
Since this was changed to `if (false)`, do we still need our patch here? (for `finishCurrentSettings()`)
cc: @samartnik @AlexeyBarabash

Reply:
We are patching only `if (false)` part, `finishCurrentSettings` change comes from the upsteam.

Reply:
ah ok - perfect 😄 Thanks!